crates/rpc-types/src/extra.rs

```diff
@@ -0,0 +1,44 @@
+use std::{
+    collections::HashMap,
+    ops::{Deref, DerefMut},
+};
+
+use serde::{Deserialize, Serialize};
+use serde_json::Value;
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+/// Wrapper allowing to catch all fields missing on the inner struct while
+/// deserialize.
+pub struct Extra<T> {
```

Comment:
i think we already have this struct: https://github.com/alloy-rs/alloy/blob/main/crates/rpc-types/src/eth/other.rs

just rename the one in that mod instead

Reply:
Not sure if I understood your comment correctly. Ended up moving `Extra<T>` to [rpc-types/src/eth/other.rs](https://github.com/alloy-rs/alloy/blob/main/crates/rpc-types/src/eth/other.rs), renaming to `WithOtherFields<T>` with a `pub other: OtherFields` field.